manifests/nodejs.yml

```
@@ -802,6 +802,8 @@ tests/:
     test_versions.py:
       Test_Events: v2.0.0
   debugger/:
+    test_debugger_code_origins.py:
+      Test_Debugger_Code_Origins:
```

I don't think this is valid yaml? It should maybe be like this?

```suggestion
      Test_Debugger_Code_Origins: missing_feature (only Fastify is supported)
```

Not sure if it should be `missing_feature` or whatever... we do support it, but it's just that we don't support it for any of the test apps in the system tests (it's only supported for Fastify, for which we don't have a test app)

If the `Test_Debugger_Code_Origins` tests were expanded to also test exit spans, we could enable this test as well though

---

Added
1f8c19cb8a463429b3bc24351edc12d193dbc65c